panel/template/base.py

```diff
@@ -449,7 +449,7 @@ def _template_resources(self):
         name = type(self).__name__.lower()
         resources = _settings.resources(default="server")
         if resources == 'server':
-            dist_path = urljoin(state.base_url, LOCAL_DIST)
+            dist_path = urljoin(state.app_url, LOCAL_DIST)
```

Comment:
Bit confused why this would be necessary (or why it would even work). `app_url` points to the URL of the specific app, e.g. `https://localhost:5006/my_app` and the dist dir is not present at that route.

Reply:
Hey @philippjfr , thanks for pointing it out, I wasn't aware of the `app_url` usage and have updated it to use state.base_url again, with a check for `startswith("/")`. 

`base_url = state.base_url[1:] if state.base_url.startswith('/') else state.base_url `

Is this fix okay?

Reply:
Looks okay. Does it work for your use cases? I'll try this out shortly but if you have already tested the jupyter proxy scenarios let me know.

Reply:
jupyter proxy scenarios work for me, and so do the examples provided in the description. Haven't tested the app based env you mentioned - `https://localhost:5006/my_app`, if there is any example to test that you can point me to, I'd be happy to test it out as well.